axelrod/strategies/memoryone.py

```diff
@@ -39,16 +45,21 @@ class MemoryOnePlayer(Player):
     with a initializing four_vector."""
 
     name = 'Generic Memory One Player'
-    memory_depth = 1
+    behaviour = {
+        'memory_depth': 1,  # Memory-one Four-Vector = (p, p, p, p)
```

Comment:
The Four-vector here is unknown (it's an abstract base class essentially). Whether it is stochastic or not is also still unknown.